src/Schema.js

```diff
@@ -337,7 +337,7 @@ define(['src/options', 'src/storage', 'src/debug'], function (CSLEDIT_options, C
 
 			// add child results to the result list
 			$.each(node.childNodes, function (i, childNode) {
-				if (childNode.localName !== null) {
+				if (childNode.nodeName !== '#text') {
```

Comment:
Is this testing if `childNode` is a text node or not? Why not test `nodeType` like below?